<launch>
  <arg name="multi_robot_name" default="tb3_2"/>
  <arg name="set_lidar_frame_id" default="base_scan"/>
  <!-- to do -->
  <arg name="x_pos" default="0"/>
  <arg name="y_pos" default="0"/>
  <arg name="yaw" default="0"/>

  <include file="$(find turtlebot3_bringup)/launch/turtlebot3_core.launch">
    <arg name="multi_robot_name" value="$(arg multi_robot_name)"/>
  </include>
  <include file="$(find turtlebot3_bringup)/launch/turtlebot3_lidar.launch">
    <arg name="set_frame_id" value="$(arg set_lidar_frame_id)"/>
    <arg name="multi_robot_name" value="$(arg multi_robot_name)"/>
  </include>

  <!-- to do -->
  <group ns = "$(arg multi_robot_name)">
      <param name="anchor0" value="tb3_0"/>
      <param name="anchor1" value="tb3_1"/>
      <param name="anchor2" value="tb3_3"/>
      <node pkg="turtlebot3_bringup" type="pozyx.py" name="turtlebot3_pozyx" output="screen" args="False $(arg x_pos) $(arg y_pos) $(arg yaw)"/>
  </group>
</launch>
